packages/models/src/organizations/organization.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(unused_variables, unused)]
```

Comment:
why did you add this line? even if no parameter is added.

Reply:
At the time, I commented on the user part and tested it, but if that part was not there, an error occurred. 
I have removed it now.